### ISSUE TITLE
sidebar is a focus element

### ISSUE THREAD

**Issue by dnoneill** [sul-dlss/sul-embed#2589]
The sidebar seems to be something you can tab to. Everything I have seen says only interactive elements should be able to be tabbed to and the sidebar is not an interactive element, it is a container for interactive elements.

https://github.com/user-attachments/assets/d7ca398f-dba0-4049-980f-fe0e57d57644

**Comment by jcoyne** [comment]
Is it possible this is because it is `role="tabpanel"`?
